starter-code/src/viking.js

```diff
@@ -1,11 +1,48 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+}
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health -= damage;
+
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+  this.name = name;
+  Soldier.call(this, health, strength);
+}
+Viking.prototype = Object.create(Soldier.prototype);
+
+Viking.prototype.attack = function() {
+  return this.strength;
```

Comment:
you don't need this prototype because you are already inheriting it from the soldier constructor.